UM/PackageManager.py

```diff
@@ -11,6 +11,9 @@
 
 from PyQt5.QtCore import pyqtSlot, QObject, pyqtSignal, QUrl, pyqtProperty
 
+from cura import ApplicationMetadata
```

Comment:
Cura shouldn't be used from Uranium, right? Are you still working on this?

Reply:
That was a mistake, it has been deleted.
The `sdk_version` is now passed from the calling cura function.